packages/components/src/DataList/components/DataListHeaderTile/components/DataListSortingOptions.module.css

```diff
@@ -37,4 +37,5 @@
 
 .option:focus-visible {
   box-shadow: var(--shadow-focus);
+  outline: transparent;
```

Comment:
❤️

Reply:
I think `Card` has this issue too, if you want to take a peek at similar instances 😄